apps/mobile/src/styles/global-styles/global-styles.ts

```diff
@@ -0,0 +1,14 @@
+import { StyleSheet } from "react-native";
+import { FLEX } from "~/styles/global-styles/flex";
+import { GAPS } from "~/styles/global-styles/gaps";
+import { MARGINS } from "~/styles/global-styles/margins";
+import { PADDINGS } from "~/styles/global-styles/paddings";
+
+const globalStyles = StyleSheet.create({
+	...FLEX,
+	...GAPS,
+	...MARGINS,
+	...PADDINGS,
+});
+
```

Comment:
Let's add borderRadius and width, height at 100% to the SIZES section.

Reply:
Don't think that we need borderRadius in globalStyles, because it is very component-specific and can vary on the shape and usage.